sourcegraph/jetbrains
# Use LogicalPosition for calculating selection range instead VisualPosition

Fixes: https://github.com/sourcegraph/jetbrains/issues/69

Additional changes: Remove unused EditorContext Java code.

Test plan:

1. Run IDE.
2. Open any code that contains some imports and methods.
3. Collapse methods visually using Ctrl + Shift + Minus.
4. View logs with `tail -f build/sourcegraph/cody-agent-trace.json`.
5. Verify that we are sending the correct selection on textDocument/didChange call.

NOTE: line position is counted from 0, not from 1.
